.env

```diff
@@ -11,5 +11,5 @@ MAX_THREADS=16
 REDIS_ENV=REDIS_URL
 REDIS_URL=redis://127.0.0.1:6379
 MEMCACHE_SERVERS=127.0.0.1:11211
-RUBYGEMS_HOST=http://rubygems.org
-DOWNLOAD_BASE=http://production.s3.rubygems.org
+RUBYGEMS_HOST=https://rubygems.org
```

Comment:
Good catch; both versions were in the code. I've consolidated to use `RUBYGEMS_URL` everywhere.